### ISSUE TITLE
Request: Easier link to GitHub Tokens page in auth flow. 

### ISSUE THREAD

**Issue by loujaybee** [gitpod-samples/Gitpod-Raycast-Extension#16]
The input placeholder for the link to GitHub when setting up the extension means I have to re-write or remember the URL it as there's no easy way to copy paste or click the placeholder text. I would suggest: 

* Make the input placeholder just look like a "real" access token, e.g. `ghp_huMrY****************************`
* Add a clickable link, or some easier flow to get redirected to GitHub for the token

**This is personal preference:** I'd remove the text "Before you can start"... etc, as it's implied by the page. Or, consider shortening the copy to be punchier / quicker? If possible could you put the "about this extension" below the input field, also? Feels like a lot of information that I'm required to read before I input the token? (just some other UI clean up thoughts, a little unrelated to the original issue, but thought I'd share if you make a change to that page 😁 !)

<img width="787" alt="image" src="https://user-images.githubusercontent.com/5528307/218056214-02885484-510a-4889-8288-a74ad6bafdda.png">

**Comment by filiptronicek** [comment]
I actually got confused and had a **Gitpod** PAT there the whole time 🤦. Maybe it's just me being silly, but making this fact more obvious would help as well!

**Comment by Palanikannan1437** [comment]
Hey @loujaybee and @filiptronicek, sorry that you had a difficult time setting it up, and thank you so much for the suggestions..these make a ton of sense, we'll make sure to make the respective changes and make things more clear!

Just a small note: This "token" is JUST a temporary fix, we're working on setting up a custom OAuth which eliminates the user to setup anything at all!

**Comment by gtsiolis** [comment]
Re-posting from a [relevant discussion](https://gitpod.slack.com/archives/C01KGM9DVRC/p1676032717697629?thread_ts=1676022750.318749&cid=C01KGM9DVRC) (internal) for visibility:

> @gtsiolis: Ah, the extension is working with an access token from GitHub, not from Gitpod. I guess others were also confused by this.

> Other issues:
> 1. It also requires a classic token which may not be clear from the instructions or UX.
> 2. You'll still get an error if you only add the minimum repo scope—this is also not clear from the instructions.

| Install instructions | Scope error |
|-|-|
| <img width="619" alt="raycast-extension-issues" src="https://user-images.githubusercontent.com/120486/218116448-3ce5930c-b5b5-42a6-985e-17e83e3be911.png"> | <img width="764" alt="ray-cart-scopes-error" src="https://user-images.githubusercontent.com/120486/218116495-2f583a33-1849-4741-82c3-189c417f74b5.png"> |

**Comment by Palanikannan1437** [comment]
Hey @gtsiolis, thanks for the quick markup...we tried to specify those scopes but it wasn't possible in Raycast (very less is customisable in this page) 

Hence we mentioned the scopes in the [installation docs](https://github.com/gitpod-samples/Gitpod-Raycast-Extension)

https://www.raycast.com/henitchobisa/gitpod

> While generating the GitHub Access Token, make sure to give the user, admin & repo access rights, else it may result in some weird behaviours.

How would you suggest adding instructions about the scopes in the "limited" above UI itself?

**Comment by gtsiolis** [comment]
> How would you suggest adding instructions about the scopes in the "limite

@Palanikannan1437 I took a quick look into Raycast's patterns and best practices while developing an extension. Here are some thoughts or potential [MVC](https://www.notion.so/gitpod/Core-Values-Differentiators-2ed4c2f93c84499b98e3b5389980992e?pvs=4#0c2e0d6c8a99404b97ccdc206b573d72) improvements:
1. I'd be specific about the scopes needed.
2. Going with the **hover title** seems like a common practice—this could be also include the scopes needed. Something like the following could suffice:
    - Your GitHub API Token (Classic). `repo` and `read:org` scope is required.
4. I'd rephrase the **input label** to _GitHub API Token (Classic)_ to make it explicit this is not asking for a token from Gitpod.
5. The input placeholder can be simpler, like _Enter your GitHub API Token (Classic)_. Using the (Classic) term in at least the input label or placeholder, if not both, sounds very helpful.

**Comment by henit-chobisa** [comment]
> > How would you suggest adding instructions about the scopes in the "limite
> 
> @Palanikannan1437 I took a quick look into Raycast's patterns and best practices while developing an extension. Here are some thoughts or potential [MVC](https://www.notion.so/gitpod/Core-Values-Differentiators-2ed4c2f93c84499b98e3b5389980992e?pvs=4#0c2e0d6c8a99404b97ccdc206b573d72) improvements:
> 
> 1. I'd be specific about the scopes needed.
> 2. Going with the **hover title** seems like a common practice—this could be also include the scopes needed. Something like the following could suffice:
>    
>    * Your GitHub API Token (Classic). `repo` and `read:org` scope is required.
> 3. I'd rephrase the **input label** to _GitHub API Token (Classic)_ to make it explicit this is not asking for a token from Gitpod.
> 4. The input placeholder can be simpler, like _Enter your GitHub API Token (Classic)_. Using the (Classic) term in at least the input label or placeholder, if not both, sounds very helpful.

Sure @gtsiolis, will try to work upon these in the upcoming [release](https://github.com/gitpod-samples/Gitpod-Raycast-Extension/milestone/1)  this week
